integration_tests/e2e/abilityToWork.cy.ts

```diff
@@ -16,14 +16,13 @@ context('Ability to work page', () => {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubAuthUser')
+    cy.task('stubPlpPrisonListPageUi')
+    cy.task('stubGetFrontEndComponents')
     cy.task('getPrisonerById')
-    cy.task('getActionPlanList')
```

Comment:
These three stubs have been removed from pretty much every test. There were there to support the original CIAG UI Prisoner List page, in that it needed to call prisoner search to get a list of prisoners for the caseload ID, PLP's action plan list API, and CIAG API's induction list.

Because this UI no longer contains a prisoner list page, and therefore the tests never visit the page, these stubs are not required.